### DISCUSSION TITLE
Nextjs Blog Post Redirects

### DISCUSSION THREAD

**Discussion by LawrenceKU03** [vercel/next.js#50287] [Help]
### Summary

Hi currently building a blog web app i have an issue,when ever i try to access a specific page on the blog through a hyperlink i get redirected back to index.js

E.g
inputting http://localhost:3000/articles/bug-sprayer-007/detail

I get redirected to http://localhost:3000 by nextjs

### Additional information

```js
This only happenes when trying to access a specific page other than the home page using a hyperlink
```

**Comment by ciruz** [comment]
Hey do you have an example repo? This can be anything.

**Reply by LawrenceKU03** [reply]
Hi @ciruz  apologies for the late reply,sadly it is private repo basically NDA so i can't but i can share the homepage,blog post page and redux store source code would that be helpful to help find the source of the bug.

**Reply by LawrenceKU03** [reply]
Hello again don't bother the bug was cause by a useEffect hook used for routing,thanks for your time
